src/sdk/model/user.ts

```diff
@@ -41,6 +44,33 @@ export class User {
     });
   }
 
+  /**
+   * Get the security for the specified user and store them into the user properties.
+   * It would actually get the list of companyInventory and roles for the specified user.
+   * Be careful when using this method, it should be called only once since it can be slow.
+   * @returns {Promise<User>}
+   */
+  async getUserWithSecurity(): Promise<User> {
```

Comment:
I thought we agreed to create a `UserWithSecurity` type to extend the `User` type with the `isUserPermitted`

Reply:
Yeah that was the first thing we said, but finally, you told me that maybe it would be ok to add this to the actual user class.
"after the user logs in, in the console we do:
`User.getCurrentUser().then(function(user) { user.getUserWithSecurity().then(function(userWithSecurity) { $rootScope.user = userWithSecurity; })})` (edited)
and we only load all of the roles/inventory when `getUserWithSecurity` is called"

Reply:
ok i see your point. I'll do it.